<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrowdChatting</class>
 <widget class="QWidget" name="CrowdChatting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="gb_recvMsg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>RecvMsg:</string>
   </property>
   <widget class="QTextBrowser" name="tb_recvMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_sendMsg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>SendMsg:</string>
   </property>
   <widget class="QLineEdit" name="le_sendMsg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_cancle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancle</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tb_send">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_userInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>CrowdInfo:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_cid">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_crowdname">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CrowdName:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_intro">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Intro:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gb_userInfo_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>191</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Online Users</string>
   </property>
   <widget class="QTextBrowser" name="tb_users">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
